react/features/app-settings/components/AppSettings.native.js

```diff
@@ -30,48 +45,58 @@ class AppSettings extends AbstractAppSettings {
      * @returns {ReactElement}
      */
     render() {
-        const { t } = this.props;
+        const { _profile, t } = this.props;
+
+        // FIXME: presentationStyle is added to workaround
+        // orientation issue on iOS
 
         return (
             <Modal
                 animationType = 'slide'
                 onRequestClose = { this._onRequestClose }
-                presentationStyle = 'fullScreen'
-                style = { styles.modal }
+                presentationStyle = 'overFullScreen'
+                supportedOrientations = { [
+                    'landscape',
+                    'portrait'
+                ] }
                 visible = { this.props._visible }>
-                <View style = { styles.headerContainer } >
+                <View
+                    style = { [
+                        styles.headerContainer,
+                        this._getSafetyPadding()
+                    ] } >
+                    <BackButton
```

Comment:
is the Android hardware back button handled by the profile settings screen ?

Reply:
Yes